solr/core/src/test/org/apache/solr/cloud/PeerSyncReplicationTest.java

```diff
@@ -289,15 +288,13 @@ private CloudJettyRunner forceNodeFailureAndDoPeerSync(boolean disableFingerprin
     indexDoc(id, docId, i1, 50, tlong, 50, t1, "document number " + docId++);
     commit();
 
-    bringUpDeadNodeAndEnsureNoReplication(replicaToShutDown, disableFingerprint);
+    bringUpDeadNodeAndEnsureNoReplication(replicaToShutDown);
 
     return replicaToShutDown;
   }
 
-  private void bringUpDeadNodeAndEnsureNoReplication(
-      CloudJettyRunner nodeToBringUp, boolean disableFingerprint) throws Exception {
-    // disable fingerprint check if needed
-    System.setProperty("solr.disableFingerprint", String.valueOf(disableFingerprint));
```

Comment:
This looks like a logic change? Suggest to defer i.e. to out-scope from this code cleanup round.

Reply:
https://github.com/apache/solr/pull/924/commits/c20b02e64f1a6b3fde9b018845cad22c683534e5 added to defer please.